### PR TITLE
CDN: offer a version without base64 encodings #469

### LINKED ISSUE #469: CDN: offer a version without base64 encodings
Similar to issue #239, we might want to make the base64 for background icons optional for CDN users by moving them out to a separate download. Or maybe creating a skin-lite.css.

### PR DESCRIPTION
**Pull request by ianmcburnie**
# Closes #469 

This is a breaking change for CDN users. Background icons will stop working.

## Goal(s)

1. Improve the performance for CDN users by making base64 data urls optional

## Changes

1. Removed base64 data urls from the main skin.css file
1. Added a new gulp task to create a new `skin-base64.min.css`
1. Updated CDN section of website

### Notes

We could, if requested, create another CDN file called `skin-full.css` which would include everything (skin and skin-base64). But I don't see a need for that yet.

I also notice that we aren't minifying `skin.css` or `docs.css` (used by our website). No idea why not! I'll create a task for this.

### CODE REVIEW

File: docs/_includes/common/cdn.html [hunk 1]
```diff
@@ -1,5 +1,7 @@
 <div id="cdn">
     <h2>CDN</h2>
-    <p>A CSS file containing the full collection of modules is available via the following url:</p>
-    <p><a href="http://ir.ebaystatic.com/cr/v/c1/skin/v6.3.2/ds4/skin.min.css">https://ir.ebaystatic.com/cr/v/c1/skin/v6.3.2/ds4/skin.min.css</a></p>
+    <p>A CSS file containing the full collection of modules (except background icon data-urls) is available via the following url:</p>
+    <p><a href="http://ir.ebaystatic.com/cr/v/c1/skin/v6.3.3/ds{{ page.ds}}/skin.min.css">https://ir.ebaystatic.com/cr/v/c1/skin/v6.3.3/ds{{ page.ds}}/skin.min.css</a></p>
+    <p>Due to their large size, background icon data-urls are considered optional, and are available separately via the following url:</p>
+    <p><a href="http://ir.ebaystatic.com/cr/v/c1/skin/v6.3.3/ds{{ page.ds}}/skin-base64.min.css">https://ir.ebaystatic.com/cr/v/c1/skin/v6.3.3/ds{{ page.ds}}/skin-base64.min.css</a></p>
```
Comment (seangates) [inline docs/_includes/common/cdn.html]: Should these URLs be changed to `v7.0.0` in anticipation? We won't publish the separate base64 version until then.

Comment (ianmcburnie) [reply docs/_includes/common/cdn.html]: Yeah, that makes sense. I'll change them now.

Comment (ianmcburnie) [reply docs/_includes/common/cdn.html]: Done.

Comment (seangates) [reply docs/_includes/common/cdn.html]: 👍

**Pull request merged**